src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1358,4 +1358,10 @@ I agree to all of the foregoing:</value>
   <data name="Naming_rules" xml:space="preserve">
     <value>Naming rules</value>
   </data>
+  <data name="Enable_power_save_mode" xml:space="preserve">
+    <value>Enable power save mode (disables all background analysis)</value>
```

Comment:
> (disables all background analysis)

:bulb: This is incorrect today, and may change in the future. To avoid confusion, it's probably best to leave it out.

```suggestion
    <value>Enable power save mode</value>
```